src/Servers/Kestrel/perf/PlatformBenchmarks/Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
```

Comment:
@sebastienros does this need to be done for the IIS projects too? https://github.com/aspnet/AspNetCore/tree/master/src/Servers/IIS/IIS/benchmarks/IIS.Performance